src/Manager.java

```diff
@@ -0,0 +1,81 @@
+
+public class Manager {
+	
+	public static void main(String args[])
+	{
+		int turns = 1;
+		
+		boolean isCaught = false;
+		Hare hare = new Hare(0,0);
+		Hound hound = new Hound(0,0);
+		hound.setDirection(hare.getDirection());
+		while(!isCaught && turns <= 100)
+		{
+			System.out.println("Turn:" +turns);
+			if(turns <=3)
+			{
+				hare.move(3, hare.getDirection());
+			}
+			else 
+			{
+				if(Math.sqrt(Math.pow(getXDisplacement(hare,hound), 2) + Math.pow(getYDisplacement(hare,hound), 2)) <=2)
+				{
+					isCaught = true;
+					break;
+				}
+					
+				if(turns % 3 == 0)
```

Comment:
NOTE: because your "turns" starts at 1, and your first 3 turns never reach this point, this condition will not trigger until the 6th turn, which is almost certainly not what you want.